be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/DbActor.scala

```diff
@@ -61,11 +87,11 @@ object DbActor extends AskPatternConstants {
   final case class WriteAndPropagate(channel: Channel, message: Message) extends Event
```

Comment:
This is outside the scope of this PR, but I'm wondering, should it be the `DbActor` responsibility to propagate messages? 🤔 (any opinion @RaulinN ?)

Reply:
Yes, it does that